src/cli/cli_commissioner.cpp

```diff
@@ -192,6 +280,24 @@ template <> otError Commissioner::Process<Cmd("joiner")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli commissioner mgmtget
+ * @code
+ * commissioner mgmtget locator sessionid
+ * Done
+ * @endcode
+ * @cparam commissioner mgmtget [locator] [sessionid] <!--
+ * -->                          [steeringdata] [joinerudpport] <!--
+ * -->                          [-x @ca{TLVs}]
+ *   * `locator`: Border Router RLOC16.
+ *   * `sessionid`: Session ID of the %Commissioner.
+ *   * `steeringdata`: Steering data.
+ *   * `joinerudpport`: UDP port of the %Joiner.
+ *   * `TLVs`: The set of TLVs to be retrieved.
+ * @par
+ * Sends a `MGMT_GET` message to the Leader.
```

Comment:
Can we explain what a `MGMT_GET` message is?  Or maybe in general, what `MGMT` means? (management?)